packages/twenty-server/src/engine/core-modules/two-factor-method/two-factor-method.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+
+import { Repository } from 'typeorm';
+
+import { TwoFactorMethod } from './two-factor-method.entity';
+
+@Injectable()
+export class TwoFactorMethodService {
+  constructor(
+    @InjectRepository(TwoFactorMethod)
+    private readonly twoFactorMethodRepository: Repository<TwoFactorMethod>,
+  ) {}
+
+  async createTwoFactorMethod(
+    userWorkspaceId: string,
+  ): Promise<TwoFactorMethod> {
+    const twoFactorMethod = this.twoFactorMethodRepository.create({
+      userWorkspace: { id: userWorkspaceId },
+    });
+
+    return this.twoFactorMethodRepository.save(twoFactorMethod);
+  }
+
+  async findAll(): Promise<TwoFactorMethod[]> {
```

Comment:
this shouldn't exist IMO:
- if this endpoint is meant to be part of the "engine api" long term (usable by business modules), I don't think we should authorize such a getter (findAll)
- if this endpoint is meant to be used within the engine, it's not useful as we can just call the ORM direclty and there is no additional business logic to wrap